core/src/main/kotlin/com/alecstrong/sql/psi/core/mysql/MySql.bnf

```diff
@@ -125,7 +125,12 @@ table_option ::= 'ENGINE' ['='] {identifier} |
   override = true
 }
 
-alter_table_rules ::= ( {alter_table_add_column} | {alter_table_rename_table} | alter_table_change_column ) {
+alter_table_rules ::= (
+  {alter_table_add_column}
+  | {alter_table_rename_table}
```

Comment:
nit: I think in most other places, we keep the pipe at the end of the previous line

Reply:
i stole it from the mysql bnf so figured it was just a bnf formatting thing

Reply:
Ha, could be, and we just do the wrong thing in this file.